README.md

```diff
@@ -1,2 +1,20 @@
 # contract-plugin
 Protoc plugins for code generation of contracts defined in grpc format
+
+## Setup & Build
+
+This project requires cmake & git submodules to build locally.
+
+```
+git clone git@github.com:AElfProject/contract-plugin.git
+cd contract-plugin
+git submodule init
+git submodule update
+
+cmake .
+make
+```
+
+## Testing
+
+the compiled `contract_csharp_plugin` binary result is written to `/opt/bin/`. You will need to copy over this binary into the `aelf.Tool` https://github.com/AElfProject/aelf-developer-tools/tree/master/aelf.tools/AElf.Tools/tools/ and build/package that C# project in-order to test your changes (i.e the output C# contract-code). Do note you'll need to build OS-specific executables for windows,linux,osx (based on the OS-folders under `aelf.tools/AElf.Tools/tools/`).
```

Comment:
Can we not mention aelf.tools here? It's not necessary to use aelf.tools.

Reply:
oh @stevenportkey  but is this `/contract_csharp_generator` binary used by any other applications? I thought only the `aelf.tools` used it for contract-generation 🤔 hence i thot it would be useful for future devs to refer to when they need to test their changes? wdyt?

Reply:
The thing is it's not necessary to use aelf.tools. We can simply use `protoc` command to test it out.